### ISSUE TITLE
Browse/search other users' profiles

### ISSUE THREAD

**Issue by mattmccutchen** [clowdr-app/clowdr-web-app#229]
I'd like to browse or search other users' profiles for similar research interests, hobbies, etc.  Trying to find all of their names and hover over each one is a pain.  If there were even a single page with all the profiles, I could use my browser's "find" command.

**Comment by bcpierce00** [comment]
Indeed, the profiles should be included in the full-text search we'd love to have!
